diagnostic.md

```diff
@@ -35,21 +35,21 @@ Using Atom, open up `diagnostic.md` and write your answers below (where indicate
 
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
- <!-- Answer Starts Here -->
+That is a relative path, as its path is being set in comparison to the current file location.
```

Comment:
it's path isn't being set in comparison to the current location.  You could type that in from any location in the file system and it would take you to the same place.